components/tidb_query_expr/src/impl_string.rs

```diff
@@ -940,6 +978,74 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_oct_string() {
+        let cases = vec![
+            (Some(b"".to_vec()), None),
```

Comment:
May be we can move the logic to later so that we dont't to repeat `Some(xxto_vec())` here？